File: sample/src/main/java/com/stefanosiano/progressimageviewsample/MainActivity.java
package com.stefanosiano.progressimageviewsample;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

import com.stefanosiano.progressimageview.ProgressImageView;
import com.stefanosiano.progressimageview.progress.PivProgressMode;

public class MainActivity extends AppCompatActivity {
    int progress = 10;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final ProgressImageView piw = (ProgressImageView) findViewById(R.id.piv);
        piw.changeProgressMode(PivProgressMode.DETERMINATE);

        final Runnable runnable = new Runnable() {
            @Override
            public void run() {
                progress += 15;
                Log.e("ASD", progress+"");
                piw.getProgressOptions().setValuePercent(progress);

                piw.postDelayed(this, 800);
            }
        };
        runnable.run();
    }
}

File: sample/src/main/java/com/stefanosiano/powerful_libraries/imageviewsample/MainActivity.kt
package com.stefanosiano.powerful_libraries.imageviewsample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.stefanosiano.powerful_libraries.imageviewsample.databinding.ActivityMainBinding

internal class MainActivity : AppCompatActivity() {

    internal lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}
